### PR TITLE
Query projectId from jira Issue-title.

### PR DESCRIPTION
**Pull request by adrian-goe**
- Add regex to match exactly the projectId pattern like `P12345`
- change order in `App.js`, so that the ID found in the service is first checked before the fallback is applied with the last project.
- Adaptation of the Atlassian test, as `service.projectId` is now a `function()`

Referencing this comment https://github.com/hundertzehn/mocoapp-browser-extension/pull/257#issuecomment-735815139

### CODE REVIEW

File: src/js/remoteServices.js [hunk 1]
```diff
@@ -1,8 +1,12 @@
 const projectRegex = /\[([\w-]+)\]/
+const projectIdRegex = /P\d{5}/gm
```
Comment (manubo) [inline src/js/remoteServices.js:2]: The convention is to have the project number in brackets, e.g. `Title [P12345]`. This is too specific for your use case, better if you can stick to this convention and remove this, OK?

**Comment by manubo** [comment]
@adrian-goe This looks good, except for the `projectIdRegex`, as I mentioned in the comment above.

**Comment by adrian-goe** [comment]
Hi, I wanted to ask what the status is here and whether I still have something to do here?

**Comment by manubo** [comment]
@adrian-goe This will go out with the next release, probably this week.

**Pull request merged**